src/app/shared/services/blog.service.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { Post } from 'src/app/shared/interfaces/post.interface';
+
+const basePostsURL: string = 'http://localhost:3000/api/posts';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class BlogService {
+  public postStream = new BehaviorSubject<Post[]>([]);
+
+  constructor (public http: HttpClient) {}
+
+  public feedPosts(): BehaviorSubject<Post[]> {
+    this.http.get<Post[]>(basePostsURL).subscribe(this.postStream);
+    return this.postStream;
+  }
```

Comment:
``return this.http.get<Post[]>(basePostsURL)``

Reply:
Thanks, it was changed in the next PR